module_1/task1/index.js

```diff
@@ -0,0 +1,10 @@
+import { Transform } from 'stream';
+
+class Reverse extends Transform {
```

Comment:
1. lets export it as `CustomReverse` or somehow prefixed to avoid conflicts with other libs.
2. lets create an additional function that will execute `process.stdin.pipe(new Reverse()).pipe(process.stdout);`

Reply:
thanks, added commit with updates